### DISCUSSION TITLE
Check --fix and format how?

### DISCUSSION THREAD

**Discussion by hyperknot** [astral-sh/ruff#8974] [General]
Is it possible to run a `check --fix` and `format` in the same command invocation? 

I have configured my IDEA to run ruff each time I save the file. Not it has to be done with 2x calls. Being able to run ruff once instead of twice would be nice.

![image](https://github.com/astral-sh/ruff/assets/494223/d22b5ae2-98c5-4eb6-927c-9af5aff3cf93)

**Comment by charliermarsh** [comment]
This isn't currently possibly but it's something we're considering! There's some discussion here: https://github.com/astral-sh/ruff/issues/8232.
